t/t2800-jobs-cmd.t

```diff
@@ -937,9 +937,15 @@ test_expect_success 'flux-jobs --format={expiration!d:%FT%T::>20.20} works' '
 	cat expiration.in | \
 	    flux jobs --from-stdin -o "{expiration!d:%b%d %R::>20.20}" \
 	    >exp-fmt.out &&
+	cat expiration.in | \
+	    flux jobs --from-stdin -o "{expiration!d:%b%d %R::>20.20h}" \
+	    >exp-fmth.out &&
 	test_debug "cat exp-fmt.out" &&
 	grep "          EXPIRATION" exp-fmt.out &&
-	grep "         $(date --date=@${exp} +%b%d\ %R)" exp-fmt.out
+	grep "         $(date --date=@${exp} +%b%d\ %R)" exp-fmt.out &&
+	test_debug "cat exp-fmt.out" &&
+	grep "          EXPIRATION" exp-fmth.out &&
+	grep "         $(date --date=@${exp} +%b%d\ %R)" exp-fmth.out
```

Comment:
i think we need to add one test for `!d` without any format being specified `{expiration!d}`, perhaps in the one above `test_expect_success 'flux-jobs --format={expiration!d:%FT%T},{t_remaining!H} works' '`

Reply:
Ok, added a separate test for that and force pushed the result.